tests/nn/data_parallel/test_fsdp_regnet.py

```diff
@@ -343,8 +339,29 @@ def test1(temp_files, ddp_ref, precision, flatten):
     fsdp_config["mixed_precision"] = precision == "mixed"
     fsdp_config["flatten_parameters"] = flatten == "flatten"
 
-    if fsdp_config["mixed_precision"] and torch_cuda_version() < (11, 0):
-        pytest.skip("Only CUDA 11 is supported with AMP equivalency")
+    if fsdp_config["mixed_precision"]:
+        # For convolution layers, we need fp32_compute_dtype. Otherwise, we run
```

Comment:
this documents the findings and best practices with conv layers. Later, can turning into some kind guide or doc.